src/main/java/teammates/ui/webapi/GetFeedbackResponsesAction.java

```diff
@@ -62,15 +99,62 @@ public JsonResult execute() {
         Intent intent = Intent.valueOf(getNonNullRequestParamValue(Const.ParamsNames.INTENT));
         FeedbackQuestionAttributes questionAttributes = logic.getFeedbackQuestion(feedbackQuestionId);
 
-        List<FeedbackResponseAttributes> responses;
+        if (questionAttributes != null) {
```

Comment:
Whether or not an entity is migrated is solely dependent on the isMigrated flag in Course.